pkgs/development/tools/misc/clojure-lsp/default.nix

```diff
@@ -18,13 +18,23 @@ stdenv.mkDerivation rec {
 
   GRAALVM_HOME = graalvm11-ce;
   CLOJURE_LSP_JAR = jar;
+  CLOJURE_LSP_XMX = "-J-Xmx4g";
 
   buildInputs = [ graalvm11-ce ];
 
   buildPhase = with lib; ''
     runHook preBuild
 
-    bash ./graalvm/native-unix-compile.sh
+    # https://github.com/clojure-lsp/clojure-lsp/blob/f3f975cd8c51fce74ffabee2678183a68791d450/graalvm/native-unix-compile.sh
+    args=("-jar" "$CLOJURE_LSP_JAR"
```

Comment:
Hum, not sure it's a good idea bring all graalvm args to here, could you just create a patch to add the liboptoonal on the .sh script?

Reply:
Done